client/scripts/views/components/header/login_selector.ts

```diff
@@ -290,6 +290,8 @@ const LoginSelector: m.Component<{
       }
     }
 
+    const hasTermsOfService = app.chain ? app.chain.meta?.chain : app.community?.meta;
```

Comment:
This doesn't seem to be checking for the terms

Reply:
`const hasTermsOfService = !!activeInvite.terms;` like in the `confirm_invite_modal` seems to be what we want here

Reply:
Haha, yes, good catch